pkg/flare/envvars.go

```diff
@@ -69,6 +69,31 @@ var allowedEnvvarNames = []string{
 	"DD_APM_ADDITIONAL_ENDPOINTS",
 	"DD_APM_PROFILING_DD_URL",
 	"DD_APM_PROFILING_ADDITIONAL_ENDPOINTS",
+	"DD_APM_WINDOWS_PIPE_BUFFER_SIZE",
+	"DD_APM_WINDOWS_PIPE_SECURITY_DESCRIPTOR",
+	"DD_APM_REMOTE_TAGGER",
+	"DD_APM_PEER_SERVICE_AGGREGATION",
+	"DD_APM_COMPUTE_STATS_BY_SPAN_KIND",
+	"DD_APM_MAX_CATALOG_SERVICES",
+	"DD_APM_RECEIVER_TIMEOUT",
+	"DD_APM_MAX_PAYLOAD_SIZE",
+	"DD_APM_LOG_FILE",
+	"DD_APM_CONNECTION_RESET_INTERVAL",
+	"DD_APM_WINDOWS_PIPE_NAME",
+	"DD_APM_SYNC_FLUSHING",
+	"DD_APM_FILTER_TAGS_REQUIRE",
+	"DD_APM_FILTER_TAGS_REJECT",
+	"DD_APM_INTERNAL_PROFILING_ENABLED",
+	"DD_APM_DEBUGGER_DD_URL",
+	"DD_APM_DEBUGGER_ADDITIONAL_ENDPOINTS",
+	"DD_APM_SYMDB_DD_URL",
+	"DD_APM_SYMDB_ADDITIONAL_ENDPOINTS",
+	"DD_APM_TELEMETRY_ENABLED",
+	"DD_APM_TELEMETRY_DD_URL",
+	"DD_APM_TELEMETRY_ADDITIONAL_ENDPOINTS",
```

Comment:
I don't think these will be needed in a flare either, since customers shouldn't have to worry about telemetry at all.